docs/dev/table/connectors/kafka.md

```diff
@@ -152,6 +159,13 @@ Connector Options
       <td>Long</td>
       <td>Start from the specified epoch timestamp (milliseconds) used in case of <code>'timestamp'</code> startup mode.</td>
     </tr>
+    <tr>
+      <td><h5>scan.topic-partition-discovery.interval</h5></td>
+      <td>optional</td>
+      <td style="word-wrap: break-word;">(disabled)</td>
```

Comment:
```suggestion
      <td style="word-wrap: break-word;">(none)</td>
```

Currentlly, we don't have `disabled` for `Default` value. We can add more explanation for the default value in Description.